libzotero/libzotero.py

```diff
@@ -26,6 +26,33 @@
 import time
 from libzotero.zotero_item import zoteroItem as zotero_item
 
+def decode_zotero_str(raw, max_iters=3):
```

Comment:
For me, this completely breaks the PDF attachments. This appears to be due to my own weird way of trying to fix the encoding. If I comment the line below out (from `LibZotero.update()`), it works again on my system.

~~~
item_attachment = item_attachment.encode('latin-1').decode('utf-8')
~~~

However, the fact that a patch that for you seems to fix things, breaks things for me makes me a bit worried. On what operating system do you work? It's not unthinkable that the encoding differs between operating systems.